federation/manifests/federation-ns.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{.FEDERATED_NAMESPACE}}
```

Comment:
yep. the alternative is that we create a `federation/manifests/default/` folder which contains the results of templating the files with "default values".